rotkehlchen/api/v1/wallet_resources.py

```diff
@@ -0,0 +1,63 @@
+from flask import Response
+from webargs.flaskparser import use_kwargs
+
+from rotkehlchen.api.v1.common_resources import BaseMethodView
+from rotkehlchen.api.v1.schemas import AddressesInteraction, NativeAssetTransfer, TokenTransfer
+from rotkehlchen.assets.asset import EvmToken
+from rotkehlchen.types import ChecksumEvmAddress, FVal, SupportedBlockchain
+
+
+class PrepareTokenTransferResource(BaseMethodView):
```

Comment:
i have nothing against separating resources but any reason in particular you did this? (learning purposes)